src/V3Os.cpp

```diff
@@ -310,6 +310,16 @@ bool V3Os::filenameIsRel(const string& filename) VL_PURE {
 #endif
 }
 
+string V3Os::fileSlashPath(const string& path) VL_PURE {
```

Comment:
```suggestion
string V3Os::filenameSlashPath(const string& path) VL_PURE {
```

Reply:
It was renamed like you initially requested. Are you sure that it is final rename?

Reply:
Yes, sorry, when saw it next to other functions I realized.